docs/root/version_history/current.rst

```diff
@@ -56,6 +56,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* compression: the :ref:`compressor <envoy_v3_api_msg_extensions.filters.http.compressor.v3.Compressor>` filter adds support for compressing request payloads. Its configuration is unified with the :ref:`decompressor <envoy_v3_api_msg_extensions.filters.http.decompressor.v3.Decompressor>` filter with two new fields for different directions - :ref:`requests <envoy_v3_api_field_extensions.filters.http.compressor.v3.Compressor.request_direction_config>` and :ref:`responses <envoy_v3_api_field_extensions.filters.http.compressor.v3.Compressor.response_direction_config>`. The latter deprecates the old response-specific fields and, if used, roots the response-specific stats in `<stat_prefix>.compressor.<compressor_library.name>.<compressor_library_stat_prefix>.response.*` instead of `<stat_prefix>.compressor.<compressor_library.name>.<compressor_library_stat_prefix>.*`.
```

Comment:
Can you add a section about the deprecated fields in the section on deprecated fields below? Thank you.

/wait

Reply:
> Can you add a section about the deprecated fields in the section on deprecated fields below? Thank you.

Added. Also fixed formatting in the message's doc strings